demo/index.html

```diff
@@ -24,9 +35,19 @@
 					text-align: center;
 				}
 			</style>
-		</custom-style>
-		<h1>Typography</h1>
-		<table>
+		</custom-style>`;
+
+document.body.appendChild($_documentContainer.content);
+</script>
+		<script type="module">
+const $_documentContainer = document.createElement('template');
+$_documentContainer.innerHTML = `<h1>Typography</h1>`;
```

Comment:
This line causes eslint to produce: `44:33  error  Strings must use singlequote  quotes`.  Not really sure why it produced that...

Reply:
It seems to have taken every direct child of the body and placed it into a template.

Reply:
We don't encounter this in the acceptance tests due to this in the eslint config `test/acceptance/*`.  Otherwise it would be complaining as well.

Reply:
Interesting, the first time we ran the modulizer against this it looked different: https://github.com/BrightspaceUI/typography/blob/polymer-3/demo/index.html

Reply:
Yeah that's more what I would expect.

Reply:
To close the loop on this - this difference is due to recent change that moved the `custom-style` into the `body`.